wordpress-mobile/WordPress-iOS
# Point WPauthenticator to the relevant branch

A draft PR to test a PR to WPAuthenticator.

The changes to WPAuthenticator that we want to test do not introduce any breaking changes to its API.

| | Before | After |
| ---- | ---- | ---- |
| iPhone | <img src="https://user-images.githubusercontent.com/2722505/101724725-37ed7800-3aea-11eb-8a32-66ef12e0bc05.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/101724743-3de35900-3aea-11eb-9347-a4117584073c.png" width="350"/> |
| iPad | <img src="https://user-images.githubusercontent.com/2722505/101724736-3b80ff00-3aea-11eb-9404-5a67ddfc6bad.png" width="350"/> | <img src="https://user-images.githubusercontent.com/2722505/101724741-3d4ac280-3aea-11eb-89cb-73f2e10e9683.png" width="350"/> |

## Changes
* Pointed WPAuthenticator to the relevant branch

## How to test
* Checkout the branch, run `bundle exec pod install`
* Log out if necessary. Notice the background of the area where the buttons in the login prologue sit
